src/web/app/svgedit/interaction/mouseInteractions.ts

```diff
@@ -193,8 +194,16 @@ let mouseSelectModeCmds;
 // but the action is not recorded until mousing up
 // - when we are in select mode, select the element, remember the position
 // and do nothing else
-const mouseDown = (evt: MouseEvent) => {
-  if (checkShouldIgnore()) return;
+const mouseDown = async (evt: MouseEvent) => {
+  if (checkShouldIgnore()) {
+    return;
+  }
+
+  // Check if the element in the clipboard can be pasted
+  workareaEvents.emit('update-context-menu', {
+    paste: await isValidNativeClipboard(),
+  });
```

Comment:
I think this can be written in `.then` format to reduce `mouseDown` execute time